tests/unit/s2n_release_non_empty_buffers_test.c

```diff
@@ -162,11 +161,14 @@ int main(int argc, char **argv)
     EXPECT_FAILURE(s2n_connection_release_buffers(conn));
 
     /* Read the rest of the buffer and expect s2n_recv to succeed */
+    n = 0;
     do {
-        ret = s2n_stuffer_recv_from_fd(&in, io_pair.server, MAX_BUF_SIZE);
+        ret = s2n_stuffer_recv_from_fd(&in, io_pair.server, MAX_BUF_SIZE, &n);
```

Comment:
@camshaft we could pass `NULL` instead of `&n`, right?

Reply:
If we're not using `n` anywhere, yeah.